INSTALL.md

```diff
@@ -12,6 +12,12 @@ of the corresponding instructions in Section 1.
 
 *We recommend that you not bother with Section 5 the first time you build RandLAPACK.*
 
+## 0. Software requirements
```

Comment:
Please add a remark that in the CMake configuration lines, some systems will have directories labeled "lib" by default while on other systems those same directories end up being called "lib64."

Reply:
Done.